lib/ddtrace/diagnostics/health.rb

```diff
@@ -0,0 +1,32 @@
+require 'ddtrace/ext/diagnostics'
+require 'ddtrace/metrics'
+
+module Datadog
+  module Diagnostics
+    # Health-related diagnostics
+    module Health
+      # Health metrics for diagnostics
+      class Metrics < ::Datadog::Metrics
+        count :api_errors, Ext::Diagnostics::Health::Metrics::METRIC_API_ERRORS
+        count :api_requests, Ext::Diagnostics::Health::Metrics::METRIC_API_REQUESTS
+        count :api_responses, Ext::Diagnostics::Health::Metrics::METRIC_API_RESPONSES
+        count :queue_accepted, Ext::Diagnostics::Health::Metrics::METRIC_QUEUE_ACCEPTED
+        count :queue_accepted_lengths, Ext::Diagnostics::Health::Metrics::METRIC_QUEUE_ACCEPTED_LENGTHS
+        count :queue_accepted_size, Ext::Diagnostics::Health::Metrics::METRIC_QUEUE_ACCEPTED_SIZE
+        count :queue_dropped, Ext::Diagnostics::Health::Metrics::METRIC_QUEUE_DROPPED
+        gauge :queue_length, Ext::Diagnostics::Health::Metrics::METRIC_QUEUE_LENGTH
+        gauge :queue_max_length, Ext::Diagnostics::Health::Metrics::METRIC_QUEUE_MAX_LENGTH
+        gauge :queue_size, Ext::Diagnostics::Health::Metrics::METRIC_QUEUE_SIZE
```

Comment:
we can drop this right?

Reply:
Yeah, I suppose we can just remove these outright too.